docs/design/openshift.md

```diff
@@ -137,7 +143,43 @@ The image field references the Update Image which contains the actual update to
 ## Update Image ##
 
 The Update Image is a container image consisting of a series of Kubernetes manifests describing the Second-Level Operators (SLOs), the `release-metadata` file required by Cincinnati, and a manifest specific to OpenShift. In the future, this image may be expanded to include other assets.
+The OpenShift manifest describes the format of the Update Image.
 
-The OpenShift manifest describes the format of the Update Image and provides a cryptographic signature of the surrounding image. The details of this manifest have not yet been finalized.
+```
+# extract payloads from a release image
+$ oc image extract quay.io/openshift-release-dev/ocp-release:4.16.15-x86_64 --path /manifests/:. --path /release-manifests/:.
+
+# examples of SLOs manifests
```

Comment:
Not everything extracted is actually SLO manifests, and the one you showed as an example are actually not (they are CVO which is the first-level operator).

I think the doc would work better if you extracted `manifests` and `release-manifests` into separate directories (not both to `.`). The manifests in `manifests` are CVO - first level operator. The manifests in `release-manifests` are SLO. The image references and metadata will be in `release-manifests` together with SLO manifests.

Reply:
Updated.